### DISCUSSION TITLE
Copilot icon missing from status bar

### DISCUSSION THREAD

**Discussion by miguelsolorio** [community/community#11684] [Copilot Conversations] [answered]
Version: 1.65.0-insider (Universal)
Commit: 86200d542474b26aa28034e0fd5789c45c747d8b
Date: 2022-02-17T07:19:54.062Z
Electron: 16.0.9
Chromium: 96.0.4664.174
Node.js: 16.9.1
V8: 9.6.180.23-electron.0
OS: Darwin arm64 21.3.0

<img width="332" alt="CleanShot 2022-02-17 at 13 30 51@2x" src="https://user-images.githubusercontent.com/35271042/154574099-14d5f0f9-a513-4b79-b108-e6411f64cc1d.png">

<img width="326" alt="CleanShot 2022-02-17 at 13 32 31@2x" src="https://user-images.githubusercontent.com/35271042/154574290-c4aa171b-530d-49dc-904b-8b4717571977.png">

Error from the console:
```
[GitHub.copilot]: 'configuration.icons' keys represent the icon id and can only contain letter, digits and minuses. They need to consist of at least two segments in the form `component-iconname`.
```

**Comment by miguelsolorio** [comment]
This appears to be because of a change int he VS Code API (see https://github.com/microsoft/vscode/issues/119101), suggestion is to update the `icons` property...

❌ From this:
```json
		"icons": [
			{
				"id": "copilot-logo",
				"description": "GitHub Copilot icon",
				"default": {
					"fontId": "copilot-font",
					"fontCharacter": "\\0041"
				}
			},
			{
				"id": "copilot-warning",
				"description": "GitHub Copilot icon",
				"default": {
					"fontId": "copilot-font",
					"fontCharacter": "\\0042"
				}
			},
			{
				"id": "copilot-notconnected",
				"description": "GitHub Copilot icon",
				"default": {
					"fontId": "copilot-font",
					"fontCharacter": "\\0043"
				}
			}
		],
```

✅ To this:
```json
		"icons": {
			"copilot-logo": {
				"description": "GitHub Copilot Icon",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0041"
				}
			},
			"copilot-warning": {
				"description": "GitHub Copilot Warning",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0042"
				}
			},
			"copilot-notconnected": {
				"description": "GitHub Copilot Not Connected",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0043"
				}
			}
		},
```

**Reply by Deekshith-Nukala** [reply]
i am also facing the same problem can  u just tell 
step by step where should do i do this
i am just a student and i started using copilot but since past 3 days i am facing this error plrase can u tell me

**Reply by miguelsolorio** [reply]
The extension needs to update itself, a user can't change this

**Comment by hsenag** [comment, marked as answer]
This should be fixed in our latest release. Apologies for the inconvenience.
